qiskit/primitives/base/base_sampler.py

```diff
@@ -80,11 +80,17 @@
 from copy import copy
 from typing import Generic, TypeVar
 
+<<<<<<< HEAD
```

Comment:
Bot screwed something up here

Reply:
This is normal behaviour when there's a true merge conflict